### DISCUSSION TITLE
Turn off or on a schedule

### DISCUSSION THREAD

**Discussion by Brouwertje** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#332] [General]
Is it possible to load the schedules in Home Assistant and turn them off or on? Some days I don't want the roborock to run the schedule inserted in the app. Would be cool if you could turn them on and off without opening the app.

**Comment by PiotrMachowski** [comment]
I don't know if it's possible. The best solution would be to move everything to HA automation's. You can also use [custom scheduler](https://github.com/nielsfaber/scheduler-card)

**Reply by Brouwertje** [reply]
True, I'll focus on that. Thank you.
